examples/async.rs

```diff
@@ -12,7 +12,7 @@ fn main() {
 }
 
 async fn run() {
-    let (eloop, http) = Http::new("http://localhost:7545").expect("transport");
+    let (eloop, http) = Http::new("http://localhost:9545").expect("transport");
```

Comment:
why this change?

Reply:
So that it works with the `truffle develop` development server which by default is on port 9545. Before `truffle-config` was manually setting a different port but I think its just better to use the defaults instead of maintaining config code.

Reply:
Currently the examples don't depend on ganache(-cli) at all, just truffle.